qa/tasks/cephfs/test_layout.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+from tasks.cephfs.cephfs_test_case import CephFSTestCase
+
+log = logging.getLogger(__name__)
+
+class TestInheritedLayout(CephFSTestCase):
```

Comment:
Nothing in `qa/suites/fs` will run this test. Add a yaml fragment to run this in `qa/suites/fs/functional/tasks`.